internal/server/GameLoop.go

```diff
@@ -267,6 +271,15 @@ func (s *Server) unaliveAgents() {
 	}
 }
 
+func (s *Server) punishBikelessAgents() {
+	for id, agent := range s.GetAgentMap() {
+		if _, ok := s.megaBikeRiders[id]; !ok {
+			// Agent is not on a bike
+			agent.UpdateEnergyLevel(utils.LimboEnergyPenalty)
```

Comment:
I think this needs a - before the arg (updateenergy adds whatever is passed in if i recall correctly)

Reply:
I added the - in the parameter definition. Would it be preferable to have it here?

Reply:
LimboEnergy is set to -0.25

Reply:
sorry I missed that, nah I guess having the word penalty is explicit enough